pkg/operator/operator.go

```diff
@@ -829,6 +829,11 @@ func (o *Operator) Config(ctx context.Context, key string) (*manifests.Config, e
 	}
 	o.userWorkloadEnabled = *c.ClusterMonitoringConfiguration.UserWorkloadEnabled
 
+	err = c.LoadEnforcedBodySizeLimit(o.client, ctx)
+	if err != nil {
+		klog.Warningf("Error loading enforced body size limit. Proceeding without it. Error: %v", err)
```

Comment:
Nothing in LoadEnforcedBodySizeLimit() takes care of resetting the value if something bad happens (e.g. if the config contains an invalid size value). This should be added.

Reply:
will use minimal body size limit in case of invalid size value.